github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/GitHubBranchTrigger.java

```diff
@@ -288,6 +283,28 @@ public void doRun(String branch) {
         return ghBranches;
     }
 
+    private List<GitHubBranchCause> mapToTriggerCauses(Set<GHBranch> remoteBranches,
```

Comment:
maybe name as `checkBranches` ? mapToTriggerCauses confused me all times i looked at this change :/